jenkins-integration/beaker/install/foss/70_install_puppet.rb

```diff
@@ -0,0 +1,26 @@
+step "Upgrade nss to version that is hopefully compatible with jdk version puppetserver will use." do
+  nss_package=nil
+  variant, _, _, _ = master['platform'].to_array
+  case variant
+  when /^(debian|ubuntu)$/
+    nss_package_name="libnss3"
+  when /^(redhat|el|centos)$/
+    nss_package_name="nss"
+  end
+  if nss_package_name
+    master.upgrade_package(nss_package_name)
+  else
+    logger.warn("Don't know what nss package to use for #{variant} so not installing one")
```

Comment:
Does this work in this pipeline?  Safer / more consistent with the rest of these beaker scripts to use `Beaker::Log.notify()` instead?  I can't remember how the magical `logger` gets hooked up but seems like we've had problems with it not being resolved in some of our pipelines in the past.